scripts/zones/Apollyon/mobs/Fir_Bholg.lua

```diff
@@ -8,60 +8,51 @@ local ID = require("scripts/zones/Apollyon/IDs")
 -----------------------------------
 local entity = {}
 
+-- Returns an ID based on raw race value.  Since gender is considered two
+-- different races for Hume, Elvaan, and Taru, and Galka and Mithra are single,
+-- we need a sane way to tell them apart.
+-- 1 = Hume, 2 = Elvaan, 3 = Taru, 4 = Mithra, 5 = Galka
+local function getRaceType(raceID)
```

Comment:
We have `getRace` and `getGender` as bindings, I wouldn't mind seeing some sort of `getGenderedRace` or something... naming is hard

Reply:
maybe a param on a single function/wrapper to returned specific id with gender or just race only? since our existing one treats each gender as its own race iID (because retail does).

Reply:
Then again, how often do we even need this? that I know of this is the 1st time.

Reply:
Might be a good thing to keep an eye on to see if it needs to be moved to a higher level.  Also, just wanted to point out that this is using a param passed from a battlefield localvar, and not from an entity itself.